README.md

```diff
@@ -31,6 +31,7 @@ First, make sure that you have the following dependencies:
   - Install [Node] (tested on v8.x.x).
   - Install [Yarn] (tested on v1.7.0).
   - Create a [GitHub API token]. No special permissions are required.
+  - Ensure that you have an SSH key (```id_rsa```) for ```git``` to use. On unix-like systems, this is usually stored in ```~/.ssh/```. If you do not, you can generate one by following these [instructions](https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent).
```

Comment:
Link text should be descriptive for accessibility; links like “this
link” or “here” or “instructions” lack context. Rephrase—perhaps:

```markdown
> If you don’t have an SSH key, you can follow [GitHub’s guide to
> SSH keys][ssh-keys] to set one up.

[ssh-keys]: https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent
```